spectator-web-spring/build.gradle

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright 2016 Google, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+dependencies {
+  compileApi project(':spectator-api')
+
+  compile 'org.springframework.boot:spring-boot-starter-actuator:1.2.8.RELEASE'
```

Comment:
Should these use compileApi? It looks like some of the spring annotations are part of the public interface.